components/Navbar.vue

```diff
@@ -48,120 +48,21 @@
-            {{ $t('classic') }}
-          </b-navbar-item>
-        </b-tooltip>
-        <template v-if="isRmrk">
-          <b-tooltip
-            label="Simplified process to create your NFT in a single step"
-            position="is-right"
-            style="display: block">
-            <b-navbar-item
-              tag="nuxt-link"
-              :to="`/${urlPrefix}/mint`"
-              data-cy="simple">
-              {{ $t('simple') }}
-            </b-navbar-item>
-          </b-tooltip>
-          <b-tooltip
-            label="AI powered process to create your NFT"
-            position="is-right"
-            append-to-body>
-            <b-navbar-item
-              tag="nuxt-link"
-              :to="`/${urlPrefix}/creative`"
-              data-cy="creative">
-              {{ $t('creative') }}
-            </b-navbar-item>
-          </b-tooltip>
-        </template>
-      </b-navbar-dropdown>
-
-      <b-navbar-dropdown
-        v-if="isBsx || isSnek"
-        id="NavStats"
-        arrowless
-        collapsible
-        data-cy="stats">
-        <template #label>
-          <span>{{ $t('stats') }}</span>
-        </template>
-        <b-navbar-item
-          tag="nuxt-link"
-          :to="`${
-            accountId
-              ? `/${urlPrefix}/offers?target=${accountId}`
-              : `/${urlPrefix}/offers`
-          }`"
-          data-cy="global-offers">
-          {{ $t('navbar.globalOffers') }}
-        </b-navbar-item>
-        <b-navbar-item
-          tag="nuxt-link"
-          :to="`/${urlPrefix}/stats`"
-          data-cy="offers-stats">
-          <span> {{ $t('navbar.offerStats') }}</span>
-        </b-navbar-item>
-        <b-navbar-item
-          tag="nuxt-link"
-          to="/series-insight"
-          data-cy="series-insight">
-          Series
-        </b-navbar-item>
-      </b-navbar-dropdown>
-      <b-navbar-dropdown
-        v-if="isRmrk"
-        id="NavStats"
-        arrowless
-        collapsible
-        data-cy="stats">
-        <template #label>
-          <span>{{ $t('stats') }}</span>
-        </template>
-        <template>
-          <b-navbar-item tag="nuxt-link" to="/spotlight" data-cy="spotlight">
-            {{ $t('spotlight.page') }}
-          </b-navbar-item>
-          <b-navbar-item
-            tag="nuxt-link"
-            to="/series-insight"
-            data-cy="series-insight">
-            Series
-          </b-navbar-item>
-          <b-navbar-item tag="nuxt-link" to="/sales" data-cy="sales">
-            Sales
-          </b-navbar-item>
-          <b-navbar-item tag="nuxt-link" to="/hot" data-cy="hot">
-            Hot
-          </b-navbar-item>
-        </template>
-      </b-navbar-dropdown>
-      <LazyChainSelect
+        class="navbar-create custom-navbar-item"
+        data-cy="create"
+        :is-rmrk="isRmrk" />
+      <NavbarStats
+        class="navbar-stats custom-navbar-item"
+        data-cy="stats"
+        :is-bsx="isBsx"
```

Comment:
if another chain is coming, will we be adding line by line? 
I guess we should look for more functional and future proof way